✏️ Fix typo in Cascade lesson (#5656)
GoogleChrome/web.dev: src/site/content/en/learn/css/the-cascade/index.md

An inline `style` attribute with CSS declared in it will override all other CSS,
regardless of its position, unless a declaration has `!important` defined.

> Position also applies to the order *inside* a CSS rule.

Reviewer: ```suggestion
Position also applies in the order of your CSS rule.
```
still clarifies, fixes the grammar, and introduces less change?

Author: agreed

Author: You want we to squash these commits?
